src/main/java/frc/robot/subsystems/vision/VisionEstimation.java

```diff
@@ -187,7 +184,7 @@ private Matrix<N3, N1> confidenceCalculator(EstimatedRobotPose estimation,
                                                                                 * VisionConstants.TAG_PRESENCE_WEIGHT)));
 
                 confidenceTelemetry.addNumber(confidenceMultiplier);
-                return visionMeasurementStdDevs.times(confidenceMultiplier);
+                return Constants.PoseEstimator.baseVisionMeasurementStdDevs.times(confidenceMultiplier);
```

Comment:
Using the constant now.

Reply:
You can just move the base stdev to VisionConstant, and use VisionConstant. to use it

Reply:
Nevermind

Reply:
Yeah, I was trying to keep `stateStdDevs` and `baseVisionMeasurementStdDevs` under the same roof (Constants.PoseEstimator), since they are related to the pose estimation, at the same time, `baseVisionMeasurementStdDevs` is related to vision... so was not sure which association is stronger... but chose the first one. Let me know if you think we should split them/rearrange them.